space_packet_parser/xtce/encodings.py

```diff
@@ -237,12 +238,12 @@ def __init__(
         self.discrete_lookup_length = discrete_lookup_length
         self.length_linear_adjuster = length_linear_adjuster
 
-    def _calculate_size(self, packet: packets.CCSDSPacket) -> int:
+    def _calculate_size(self, packet: spp.SpacePacket) -> int:
         """Calculate the size of the raw string buffer field
 
         Parameters
         ----------
-        packet : packets.CCSDSPacket
+        packet : Packet
```

Comment:
```suggestion
        packet : SpacePacket
```

Reply:
Ok there's a lot of these. I'll leave it to you to fix those typehints

Reply:
Yes, sorry I guess I was only doing my find+replace over code before and missed the docstring types. Thanks for pointing it out!